ci/deploy_tidb_operator_staging.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020. PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.transform.Field
+
+@Field
+def values = '''
+controllerManager:
+  replicas: 2
+scheduler:
+  replicas: 2
+admissionWebhook:
+  create: true
+  hooksEnabled:
+    statefulSets: true
```

Comment:
StatefulSets webhook is used for canary upgrading. Is it necessary to be enabled in the default?

Reply:
IMO, yes, we have a section about canary release in the IDC manual